### ISSUE TITLE
[Dropdown] [A11Y] Accessability violations in Carbon Dropdown

### ISSUE THREAD

**Issue by JordanWSmith15** [carbon-design-system/carbon-addons-iot-react#1469]
### Describe the bug

Accessibility scanner is picking up violations in the dropdown component. We cannot fix these in the Graphite repo. 

### To Reproduce

1. Go to '.https://carbon-addons-iot-react.com/?path=/story/dropdown--default'
2. Click on 'Accessibility' tab
3. See error

![Screen Shot 2020-08-06 at 12 19 24 PM](https://user-images.githubusercontent.com/21066152/89556309-29f7b580-d7df-11ea-9b4b-f1c536165809.png)

Relates to:
https://jsw.ibm.com/browse/GRAPHITE-15962
https://jsw.ibm.com/browse/GRAPHITE-15958

**Comment by tay1orjones** [comment]
Should be fixed via #1323. As part of that upgrade Carbon updated the downshift dependency which defers the rendering of the menu until the dropdown is actually clicked/opened. The storybook a11y checks are passing in the PR #1462 branch:

![image](https://user-images.githubusercontent.com/3360588/89817594-0fd71380-db0e-11ea-8505-f952f9c97b66.png)
